allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureParameterizedTest.java

```diff
@@ -12,13 +12,11 @@
 public class AllureParameterizedTest {
```

Comment:
I have put two another comments below regarding simplifying the examples – removing anything not directly related to the "parametrization" context.

I am posting this comment here as my final proposal, that also includes removing of `@DisplayName` that is also not directly related to the context of this test - we already have the corresponding `AllureNameTest.java`

So, I would simplify to: 

```
public class JUnit5ParameterizedTest {

    @ParameterizedTest(name = "{displayName} [{argumentsWithNames}]")
    @ValueSource(strings = {"John", "Mike"})
    public void allureParameterizedTest(String name) {
        parameter("[added dynamically] Name", name);
        step("testing somebody called: " + name);
    }

}
```

Yet I might miss something... maybe all this is oversimplified:D